storage/cloud-client/acl_test.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2016 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gcloud import storage
+import gcloud.storage.acl
+import pytest
+
+import acl
+
+# Typically we'd use a @example.com address, but GCS requires a real Google
+# account.
+TEST_EMAIL = 'jonwayne@google.com'
```

Comment:
I could be part of `cloud_config`, but it's not really worth the effort. Let me see if `nobody@google.com` or 'noreply@google.com` will work.